CHANGELOG.md

```diff
@@ -28,6 +28,126 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v7.0.0-beta5](https://github.com/shopsys/shopsys/compare/v7p.0.0-beta4...v7.0.0-beta5) - 2019-01-17
+
+### [shopsys/coding-standards]
+
+#### Changed
+
+- [#707] order of imports and sort algorithm is now explicitly set
+
+### [shopsys/framework]
+
+#### Added
+
+- [#619] Simplify adding new form groups
+- [#723] Copy categories to variant if main variant is edited or variant is added
+
+#### Changed
+
+- [#688] renamed AdvancedSearchFacade to AdvancedSearchProductFacade
+- [#650] change visibility in extendable classes according to our docs
+- [#692] redirectController is now registered as a public container service
+- [#624] ImageUploadType now can be used for uploading specific image types
+- [#690] refactored checking duplicated e-mail to User constructor
+- [#712] Stop generating url for unsafe route 'fp_js_form_validator.check_unique_entity'
+- [#687] refactored AdvancedSearch filter names to Filter classes
+
+#### Fixed
+
+- [#706] fixed symfony deprecation warnings
+- [#647] fixed link to our Twitter account [@ShopsysFW] in demo data
+- [#640] Fixed cart merging after login (closes [#588]), Thanks to [@dzarda86]
+- [#628] selectboxes rollout problem for dev env
+- [#644] JsFormValidatorFactory: make sure prototype's parent is set in CollectionType
+- [#655] calculatedHidden attribute of a main variant is taken over from the hidden attribute, Thanks to [@dzarda86]
+- [#676] formatNumber filter supplied with missing orderLocale variable, Thanks to [@dzarda86]
+
+#### Removed
+
+- [#641] FormGroup: unused option "is_group_container_to_render_as_the_last_one" removed
+- [#609] remove redundant PaginationResult instantiation
+
```

Comment:
You've omitted #567, was that intentional? I would include it in `shopsys/framework`'s *Fixed* section